src/pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -168,6 +173,9 @@ def render(self, task) -> Text:
                 text += f"{k}: {round(v, 3) if isinstance(v, float) else v} "
             return Text(text, justify="left", style=self._style)
 
+else:
+    Task, Style = None, None  # type: ignore[assignment, misc]
```

Comment:
```suggestion
    Task, Style = Any, Any  # type: ignore[assignment, misc]
```
Please, let's not use `None` in places, where we need types, otherwise, ugly things can happen (https://github.com/Lightning-AI/lightning/pull/14809)